java/java.lsp.server/vscode/package.json

```diff
@@ -648,10 +656,14 @@
 				},
 				{
 					"command": "nbls:Database:netbeans.db.explorer.action.Connect",
-					"when": "viewItem =~ /class:ddl.DBConnection/"
+					"when": "viewItem =~ /class:org.netbeans.api.db.explorer.DatabaseConnection/"
 				},
 				{
 					"command": "nbls:Database:netbeans.db.explorer.action.Disconnect",
+					"when": "viewItem =~ /class:org.netbeans.api.db.explorer.DatabaseConnection/"
+				},
+				{
+					"command": "nbls:Database:netbeans.db.explorer.action.Refresh",
```

Comment:
Q: should the `ddl.DBConnection` remain here, or should it be replaced with `DatabaseConnection` as well ?

Reply:
It should remain here as ddl.DBConnections is on all other nodes where Refresh action is needed